modules/introduction/pages/developer-preview/collections/cbstats-reference/cbstats-collections-details.adoc

```diff
@@ -0,0 +1,92 @@
+= collections-details
+:page-topic-type: reference
+
+[abstract]
+Provides low-level details on collections.
+
+== Syntax
+
+----
+cbstats host:11210 [common options] collections-details [vbucket-reference]
+----
+
+== Description
+
+This command is used to provide low-level details on collections, for a specified bucket.
+This includes details identifying the existing collections; and vBucket-specific details, which include sequence numbers and item counts.
+
+== Options
+
+The optional `vbucket-reference` parameter is an integer, in the range of `0` to `1023` inclusive, specifying a particular vBucket.
+If a vBucket is specified, the only details returned are the vBucket-specific details for that vBucket.
+If no vBucket is specified, vBucket-specific details for all vBuckets are returned, as well as details identifying the existing collections for the bucket.
+
+For common [.cmd]`cbstats` options, see xref:cli:cbstats-intro.adoc[cbstats].
+
+== Examples
+
+The following command retrieves collections-details on all vBuckets for `testBucket`:
+
+----
+/opt/couchbase/bin/cbstats -u Administrator -p password -b testBucket \
+localhost:11210 collections-details
+----
+
+The initial section of the output is as follows:
+
+----
+manifest:collection:0x0:name:                    _default
+manifest:collection:0x8:name:                    my_collection_in_my_scope_1
+manifest:collection:0x9:name:                    my_collection_in_my_scope_2
+manifest:collection:0xa:name:                    my_collection_in_default_scope
+manifest:collections:                            4
+manifest:default_exists:                         true
+manifest:uid:                                    4
+vb_0:collection:0x0:entry:end_seqno:             -6
```

Comment:
Tony, I mentioned this output would be changing, the following is an example of the current output that will ship in the Developer Preview


`manifest:collection:0x0:name:                       _default`
`manifest:collection:0x8:name:                       my_collection_in_my_scope_1`
`manifest:collection:0x9:name:                       my_collection_in_my_scope_2`
`manifest:collection:0xa:name:                       my_collection_in_default_scope`
`manifest:collections:                               4`
`manifest:default_exists:                            true`
`manifest:uid:                                       4`
`vb_0:collection:0x0:entry:high_seqno:              26`
`vb_0:collection:0x0:entry:items:                   120`
`vb_0:collection:0x0:entry:persisted_high_seqno:    26`
`vb_0:collection:0x0:entry:scope:                   10`
`vb_0:collection:0x0:entry:start_seqno:             26`
.
.
.

Reply:
basically end_seqno is removed and high_seqno is added

Reply:
I've modified the examples based on the latest output, from build 3506.